consensus/pbft/execute.go

```diff
@@ -98,6 +99,7 @@ func (r *Replica) execute(view uint, sequence uint, digest string) error {
 			RequestTimestamp: request.Timestamp,
 			Replica:          r.id,
 		},
+		Signature: hex.EncodeToString(res.Signature),
```

Comment:
PBFT already has and mandates signatures - note the `msg.Sign()` invocation below that will overwrite this value.